tools/runqlen.bt

```diff
@@ -33,7 +33,7 @@ profile:hz:99
 {
 	$task = (struct task_struct *)curtask;
 	$my_q = (struct cfs_rq *)$task->se.cfs_rq;
-	$len = $my_q->nr_running;
+	$len = (int64)$my_q->nr_running;
```

Comment:
What would happen if the typecast was not added here? An error or an incorrect value assigned to `$len`?

Reply:
Ok, makes sense. But maybe this would be a candidate for the new "Breaking" changelog section as it may cause previously working scripts fail.

Reply:
My hope is there aren't many of these cases but I can add an entry if you prefer.

Reply:
Yeah, I'm not sure either. Let's leave it without an entry for now and discuss on the office hours how we want to treat breaking changes in general.